pkg/scheduler/framework/runtime/framework.go

```diff
@@ -528,7 +543,7 @@ func fillEventToPluginMap(p framework.Plugin, eventToPlugins map[framework.Clust
 	// We treat it as: the plugin is not interested in any event, and hence pod failed by that plugin
 	// cannot be moved by any regular cluster event.
 	if len(events) == 0 {
-		klog.InfoS("Plugin's EventsToRegister() returned nil", "plugin", p.Name())
```

Comment:
Why remove the plugin name key:value pair? Is the plugin name already set some place else?